loading.bs

```diff
@@ -331,6 +390,433 @@ Note: Applying the signed exchange's response here has the effect of letting a
+                |link|'s [=Target Attributes list|Target Attributes=] whose
+                first item matches the string `"variant-key"` or the empty
+                string ("") if it is not present.
+            1. Let |headerIntegrity| be the second item of the first tuple of
+                |link|'s [=Target Attributes list|Target Attributes=] whose
+                first item matches the string `"header-integrity"` or continue
+                if it is not present.
+            1. [=list/Append=] (|linkTarget|, |linkVariants|, |linkVariantKey|,
+                |headerIntegrity|) to |allowedSxgLinks|.
+        1. Let |preloadLinkItems| be an empty [=list=].
+        1. Let |canLoadAlternateSxg| be true. 
+        1. For each |link| of |links|:
+            1. If |link|'s [=Relation Type=] is not 'preload', then continue.
+            1. Let |linkTarget| be |link|'s [=Link Target=].
+            1. Let |asAttribute| be the second item of the first tuple of
+                |link|'s [=Target Attributes list|Target Attributes=] whose
+                first item matches the string `"as"` or the empty string ("") if
+                it is not present.
+            1. If |asAttribute| is an empty string, then continue.
+            1. If |asAttribute| is `"image"`, then:
+                1. Let |imagesrcset| be the second item of the first tuple of
+                    |link|'s [=Target Attributes list|Target Attributes=] whose
+                    first item matches the string `"imagesrcset"` or the empty
+                    string ("") if it is not present.
+                1. Let |imagesizes| be the second item of the first tuple of
+                    |link|'s [=Target Attributes list|Target Attributes=] whose
+                    first item matches the string `"imagesizes"` or the empty
+                    string ("") if it is not present.
+                1. Create a <{link}> element |linkElement| which <{link/href}>
+                    attribute is |linkTarget|, and <{link/imagesrcset}>
+                    attribute is |imagesrcset|, and <{link/imagesizes}> is
+                    |imagesizes|.
+                1. Let |selected source| and <var ignore=''>selected pixel
+                    density</var> be the URL and pixel density that results from
+                    [=selecting an image source=] given |linkElement|,
+                    respectively.
+                1. If |selected source| is not null, then set |linkTarget| to
+                    the result of [=URL parser|parsing=] |selected source|, with
+                    a base URL of |navigationParams|'s [=navigation
+                    params/response=]'s [=response/URL=].
+            1. Let |headerIntegrity| be an empty string.
+            1. For each |allowedSxgLink| of |allowedSxgLinks|:
+                1. If the first element isn't the same as |linkTarget|, then
+                    continue.
+                1. Let |innerLinkVariants| be the second item of
```

Comment:
Similarly down here, because [tuples are structs](https://infra.spec.whatwg.org/#tuple), all the items have names, which need to be implied or stated when the tuple is created. Here, the variable names that created the tuple work well as names, so you can just use `|allowedSxgLink|'s innerLinkVariants` or similar wherever it's used, instead of needing steps to pull them out into local variables.